ci/ansible/roles/pulp-backup/defaults/main.yaml

```diff
@@ -0,0 +1,20 @@
+---
+# location of backup on remote
+fetch_to_localhost: true
+remote_bkup_dir: "{{ ansible_user_dir }}/remote_backups"
+local_bkup_dir: "local_backups"
+files_to_backup:
+  etc_pulp:
+    src_dir: /etc/pulp
+    file: etc_pulp
+  etc_pki_pulp:
+    src_dir: /etc/pki/pulp
+    file: etc_pki_pulp
+  var_lib_pulp:
+    src_dir: /var/lib/pulp
+    file: var_lib_pulp
+
+db_to_bkup:
+  mongodb:
```

Comment:
Yeah that is reasonable. At one point I had it like this but then changed it to be more like the files "in case" there was some other database we would want to back up. But that is probably never going to happen so I agree this is a good change.